app/middleware/searchProcessor.js

```diff
@@ -0,0 +1,13 @@
+function stripWhitespace(req, res, next) {
```

Comment:
The name of this module i.e. `searchProcessor` isn't a very accurate description of what it is doing. Perhaps `stripWhitespace` would be better and help other people understand what it is doing?